doc/advanced_setup.rst

```diff
@@ -103,6 +103,29 @@ runtime accordingly.
 If you use another Python setup and you encounter some difficulties please
 report them on the MNE mailing list or on github to get assistance.
 
+Installing Mayavi
+^^^^^^^^^^^^^^^^^
+
+Mayavi is only available for Python2.7. If you have Anaconda installed (recommended), the easiest way to install `mayavi` is to do::
+
+    conda install mayavi
```

Comment:
I still think we should add `$` to the `bash` lines even if we haven't been consistent before. I'm okay omitting the `>>>` from Python lines if we have to.